FirebaseVertexAI/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [feature] The Firebase Vertex AI SDK now builds in Swift 6 mode.
```

Comment:
nit: maybe write this as "... no longer requires `@preconcurrency` when imported in Swift 6."

Reply:
Done. Thanks